plugins/optimization-detective/detection.php

```diff
@@ -127,8 +129,10 @@ function od_get_detection_script( string $slug, OD_URL_Metric_Group_Collection $
 		'urlMetricHMAC'          => od_get_url_metrics_storage_hmac( $slug, $current_etag, $current_url, $cache_purge_post_id ),
 		'urlMetricGroupStatuses' => array_map(
 			static function ( OD_URL_Metric_Group $group ): array {
+				$max = $group->get_maximum_viewport_width();
 				return array(
-					'minimumViewportWidth' => $group->get_minimum_viewport_width(),
+					'minimumViewportWidth' => $group->get_minimum_viewport_width(), // Exclusive.
+					'maximumViewportWidth' => PHP_INT_MAX === $max ? null : $max, // Inclusive.
```

Comment:
As mentioned above, if we allow `maximum_viewport_width` to be `null` as another way of saying "infinity" them this logic wouldn't be needed.